Added og: image, og:url and robots meta tags to help SEO

nautatva opened the pull request:
Fix for https://github.com/webgems/webgems/issues/151 opengraph tags are added. Also added a point in PHP and icons

lostdesign commented on resources/php.json:
```diff
@@ -67,6 +67,12 @@
       "tags": []
+    },
+    {
+      "title": "Stuck at a code? Let's find the problem",
```
This resource does not add much value to be honest. Linting should be done in the IDE and not by pasting it on some website. Can you please remove this?

nautatva commented:
Summary of the commit:
- Made the description to tell a little more  about webDevs, also includes keywords for SEO
- Removed phpCodeChecker
- Removed the extra tag from fontello
- Linted the code properly
- Added SEO tags which now include title and description as well
- Referenced the title and description to a common package.json
